src/Emulator/Instructions/Instructions.js

```diff
@@ -18,18 +16,13 @@ const Instructions = ({ panelData }) => {
       <div>
         <h4 className="tasks-title">Tasks</h4>
         <ol>
-          {panelData.tasks.map(task => <li>{task}</li>)}
-          {/* {panelData.tasks.map(task => <li>{task}</li>)}
-          {panelData.tasks.map(task => <li>{task}</li>)}
-          {panelData.tasks.map(task => <li>{task}</li>)}
-          {panelData.tasks.map(task => <li>{task}</li>)}
-          {panelData.tasks.map(task => <li>{task}</li>)}
-          {panelData.tasks.map(task => <li>{task}</li>)}
-          {panelData.tasks.map(task => <li>{task}</li>)} */}
+          {panelData.tasks.map((task) => (
+            <li>{task}</li>
+          ))}
         </ol>
```

Comment:
A snapshot would be great here.

Render your component with some specific `panelData` props to ensure that snapshots match in the future and that everything renders correctly